smg-automotive/components-pkg
# fix: changed chevron icon  only for checkbox group

References https://smg-au.atlassian.net/browse/DM-3106

## Motivation and context

Better visibility in the chevron icon on the checkbox group

## Take into consideration

* I checked with Sancho to be 100% sure if this change will be ONLY applied on the checkbox group and he confirmed this change is ONLY here. The other chevron icons like on the select component is another topic that he needs to check with the designers.

* I checked with Sancho and the chevron icon on the opening hours on PDP is correct what we have in production. So, no change here as well

## Before

![Screenshot 2024-08-23 at 12 01 17](https://github.com/user-attachments/assets/bba80f2b-7639-41b0-b885-dea639b8c7e6)

## After

![Screenshot 2024-08-23 at 12 01 57](https://github.com/user-attachments/assets/b7a05f6a-deae-45e4-8e4d-68b2d50f0278)


## How to test

Please check that this change only applies on the checkbox group in our library.
Here is how it looks in listings-web (please check the checkbox group on advanced search and srp pages) ->  https://fix-chevron-checkboxgroup-listings-web.branch.autoscout24.dev/de/s

## Thank you 🥔
